pkg/output/file_csv.go

```diff
@@ -13,7 +13,11 @@ var staticheaders = []string{"url", "redirectlocation", "position", "status_code
 
 func writeCSV(filename string, config *ffuf.Config, res []ffuf.Result, encode bool) error {
 	header := make([]string, 0)
-	f, err := os.Create(filename)
+	extension := ".csv"
+	if encode {
+		extension = ".ecsv"
+	}
+	f, err := os.Create(config.OutputFile + extension)
```

Comment:
I get your point but your commenting the wrong lines here. CSV needs the if statement because the same method is printing .csv and .ecsv files. Nevertheless, I've implemented a buildFileName method inside `output.go` (see current commit).